### PR TITLE
[#55112] Rename Storages::ManageNextcloudIntegrationJob

### PR DESCRIPTION
**Pull request by ba1ash**
https://community.openproject.org/work_packages/55112

### CODE REVIEW

File: modules/storages/app/workers/storages/manage_storage_integrations_job.rb [hunk 1]
```diff
@@ -44,12 +44,11 @@ class ManageNextcloudIntegrationJob < ApplicationJob
     good_job_control_concurrency_with(
       total_limit: 2,
       enqueue_limit: 1,
-      perform_limit: 1,
-      key: "ManageNextcloudIntegrationJob"
```
Comment (ba1ash) [inline modules/storages/app/workers/storages/manage_storage_integrations_job.rb:48]: This `key` has been removed intentionally. If there is no `key` specified GoodJob will derive it from class name.

Comment (cbliard) [reply modules/storages/app/workers/storages/manage_storage_integrations_job.rb:48]: And is it ok if `perform_limit` is removed as well? It means the [limit being used will be `total_limit`](https://github.com/bensheldon/good_job/blob/e632cf3f60adb5b59d0b6338d4b109387b9bc0d5/lib/good_job/active_job_extensions/concurrency.rb#L71), which is 2. Correct?

**Review by cbliard — APPROVED** [review]
Thanks for doing the renaming.
I only have a tiny question about the number of concurrent performing jobs.
Once it is all clear, feel free to merge the PR

File: modules/storages/app/workers/storages/manage_storage_integrations_job.rb (hunk 1 above)
Comment (ba1ash) [reply modules/storages/app/workers/storages/manage_storage_integrations_job.rb:48]: It is not okay to remove it and it is not removed. Only comma afterwards is gone.

Comment (cbliard) [reply modules/storages/app/workers/storages/manage_storage_integrations_job.rb:48]: ah sorry, I missed it as the comment was onyl on the deleted lines. Good 👍

**Pull request merged**